share/completions/castnow.fish

```diff
@@ -0,0 +1,30 @@
+set -l __fish_castnow_keys "space\tToggle\ between\ play\ and\ pause m\tToggle\ mute t\tToggle\ subtitles up\tVolume\ up down\tVolume\ down left\tSeek\ backward right\tSeek\ forward n\tNext\ in\ playlist s\tStop\ playback quit\tQuit"
```

Comment:
I can see why this worked...

`mount.fish` uses single quotes in `complete` `-a` argument.
```fish
complete -c mount -x -s o --description 'Mount option' -a '(__fish_append , $__fish_mount_opts)'
```
So `$__fish_mount_opts` doesn't get replaced before being passed to `complete`.

Whereas I've used double quotes in `complete`:
```fish
complete -c castnow -l command -d "Execute key command(s)" -x -a "(__fish_append , $__fish_castnow_keys)"
```
So here, `$__fish_castnow_keys` get replaced with its value before being passed to `complete`.

The advantage here is that I can define `__fish_castnow_keys` as a local variable (and not a global one as in `mount.fish`).

So what would you prefer?